Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/contest-card/ContestCard.tsx

```diff
@@ -4,12 +4,15 @@ import Countdown from '../../guidelines/countdown/Countdown';
 import { convertToSecondsRemaining } from '../../../utils/dates';
 import styles from './ContestCard.module.scss';
 import { LinkButton } from '../../guidelines/buttons/Button';
+import concatClassNames from '../../../utils/class-names';
 
 interface IContestCardProps {
     contest: IIndexContestsType
 }
 
 const ContestCard = ({ contest }: IContestCardProps) => {
+    const cardsContests = 'all-cards-contests';
```

Comment:
1. Why the plural? Is this not a **single** contest card?
2. `contestCardClassName`